images/dockerregistry/config.yml

```diff
@@ -58,3 +44,13 @@ openshift:
       maxrunning: 0
       maxinqueue: 0
       maxwaitinqueue: 0
+  quota:
+    enabled: false
+  cache:
+    blobrepositoryttl: 10m
+    quotattl: 1m
```

Comment:
Move quotattl to the quota section.
/cc @miminar

Reply:
@dmage @miminar done